tox.ini

```diff
@@ -53,14 +58,23 @@ commands_pre =
   pip install -e {toxinidir}/opentelemetry-sdk
   pip install -e {toxinidir}/ext/opentelemetry-ext-wsgi
   pip install -e {toxinidir}/ext/opentelemetry-ext-http-requests
+  pip install -e {toxinidir}/opentelemetry-example-app
 
 commands =
 ; Prefer putting everything in one pylint command to profit from duplication
 ; warnings.
-  black --check --diff opentelemetry-api/src/opentelemetry opentelemetry-api/tests/ opentelemetry-sdk/src/opentelemetry opentelemetry-sdk/tests/ ext/opentelemetry-ext-http-requests/src/ ext/opentelemetry-ext-http-requests/tests/ ext/opentelemetry-ext-wsgi/tests/
-  pylint opentelemetry-api/src/opentelemetry opentelemetry-api/tests/ opentelemetry-sdk/src/opentelemetry opentelemetry-sdk/tests/ ext/opentelemetry-ext-http-requests/src/ ext/opentelemetry-ext-http-requests/tests/ ext/opentelemetry-ext-wsgi/tests/
-  flake8 opentelemetry-api/src/ opentelemetry-api/tests/ opentelemetry-sdk/src/ opentelemetry-sdk/tests/ ext/opentelemetry-ext-wsgi/src/ ext/opentelemetry-ext-wsgi/tests/ ext/opentelemetry-ext-http-requests/src/
-  isort --check-only --diff --recursive opentelemetry-api/src opentelemetry-sdk/src ext/opentelemetry-ext-wsgi/src ext/opentelemetry-ext-wsgi/tests ext/opentelemetry-ext-http-requests/src/
+  black --check --diff .
+  pylint opentelemetry-api/src/opentelemetry \
+         opentelemetry-api/tests/ \
+         opentelemetry-sdk/src/opentelemetry \
+         opentelemetry-sdk/tests/ \
+         ext/opentelemetry-ext-http-requests/src/ \
+         ext/opentelemetry-ext-http-requests/tests/ \
+         ext/opentelemetry-ext-wsgi/tests/ \
+         opentelemetry-example-app/src/opentelemetry_example_app/ \
+         opentelemetry-example-app/tests/
+  flake8 .
+  isort --check-only --diff --recursive .
```

Comment:
Can isort properly distinguish internal and external imports with that (or does it matter at all)?

Reply:
A little digging here, but it does look like it matters:

https://isort.readthedocs.io/en/latest/#how-does-isort-work

But this shouldn't be affected by the directory it runs as because:

https://github.com/timothycrosley/isort/issues/704#issuecomment-414139963

As isort determines third-part vs stdlib via importing the code, which relies on the package to be installed via tox, this shouldn't be an issue.

Also all existing import orders are unchanged.